cmd/src/server.rs

```diff
@@ -541,6 +541,18 @@ impl TiKVServer {
             None
         };
 
+        // The `DebugService` and `DiagnosticsService` will share the same thread pool
+        let debug_thread_pool = Arc::new(
+            Builder::new()
+                .threaded_scheduler()
+                .thread_name(thd_name!("debugger"))
+                .core_threads(1)
+                .on_thread_start(|| tikv_alloc::add_thread_memory_accessor())
+                .on_thread_stop(|| tikv_alloc::remove_thread_memory_accessor())
+                .build()
+                .unwrap(),
```

Comment:
Could we share grpc thread pool with debug?

Reply:
Can you explain and give more details?

Reply:
Oh, I made a mistake. This is a task pool rather rpc pool.